lib/rs/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "digitalarsenal-standards"
+version = "0.1.0"
+edition = "2021"
+description = "Space data standards framework based on CCSDS standards and Google FlatBuffers."
+license = "Apache-2.0"
+authors = ["sds@digitalarsenal.io", "shanebenlolo@gmail.com"]
```

Comment:
feel free to remove my name here if you don't want it

Reply:
Thanks for the pull!

You will absolutely be added to the list of contributors, anyone that contributes to open source should of course be recognized.

The publish script should be put in “scripts”, and the source needs to remain where it is, as it is dynamically generated by the